internal/kafka/internal/services/kafka.go

```diff
@@ -214,13 +235,6 @@ func (k *kafkaService) RegisterKafkaJob(kafkaRequest *dbapi.KafkaRequest) *error
 	// we need to pre-populate the ID to be able to reserve the quota
 	kafkaRequest.ID = api.NewID()
 
-	instanceType, err := k.DetectInstanceType(kafkaRequest)
-	if err != nil {
-		return err
-	}
-
-	kafkaRequest.InstanceType = instanceType.String()
-
 	hasCapacity, err := k.HasAvailableCapacityInRegion(kafkaRequest)
```

Comment:
I'm confused. Why this has been deleted?

Reply:
because instance type is assigned in the handler

Reply:
Got it. However if this method is not used to detect the instance type anymore, it should be renamed

Reply:
renamed